charts/console/Chart.yaml

```diff
@@ -36,7 +36,7 @@ kubeVersion: ">= 1.25.0-0"
 
 icon: https://images.ctfassets.net/paqvtpyf8rwu/3cYHw5UzhXCbKuR24GDFGO/73fb682e6157d11c10d5b2b5da1d5af0/skate-stand-panda.svg
 sources:
-  - https://github.com/redpanda-data/helm-charts
+  - https://github.com/redpanda-data/redpanda-operator/charts
```

Comment:
These links don't resolve as they need the ref set. https://github.com/redpanda-data/redpanda-operator/tree/main/charts

Should we go ahead and link to the specific folder or the chart as well? e.g. https://github.com/redpanda-data/redpanda-operator/tree/main/charts/console